lib/sequences/07_argonaut_search_sequence.rb

```diff
@@ -166,7 +167,13 @@ def validate_reply(klass, reply)
           'http://www.fhir.org/guides/argonaut/r2/Conformance-server.html',
           'All servers SHALL make available the read interactions for the Argonaut Profiles the server chooses to support.' do
 
-    todo
+    assert !@allergyintolerance.nil?, 'Expected valid DSTU2 AllergyIntolerance resource to be present'
+    id = @allergyintolerance.try(:id)
+    assert !id.nil?, "AllergyIntolerance id not returned"
+    allergyintolerance_read_response = @client.read(FHIR::DSTU2::AllergyIntolerance, id)
```

Comment:
Should we pull these out into their own function, like we are doing for `validate_reply()` for the searches?  And speaking of, we probably should rename `validate_reply()` to `validate_search_reply()`.